config/sync/shortcut.set.default.yml

```diff
@@ -1,4 +1,4 @@
-uuid: NULL
+uuid: d33bd571-9d49-41d0-9713-fa09d806c25c
```

Comment:
Be careful not to commit this file in the future, it re-introduces the core bug involving shortcuts during site installation. In the advanced git lesson we'll learn how to more surgically choose what to commit.